packages/reapit-connect-management/src/services/user.ts

```diff
@@ -35,24 +36,49 @@ export const updateUser = async (user: UpdateUserParams, userId: string): Promis
   }
 }
 
-export const updateUserGroup = async (user: UpdateUserGroupParams, userGroupId: string): Promise<any | undefined> => {
+export const addMemberToGroup = async (group: UpdateUserGroupParams): Promise<any | undefined> => {
   try {
     const session = await reapitConnectBrowserSession.connectSession()
 
     if (!session) throw new Error('No Reapit Connect Session is present')
 
     const api = window.reapit.config.platformApiUrl
-    const url = `${URLS.USERS_GROUPS}/${userGroupId}`
+    const url = `${URLS.USERS_GROUPS}/${group.id}/members`
     const path = `${api}${url}`
 
     const response = await fetch(path, {
-      method: 'PUT',
+      method: 'POST',
       headers: {
         ...BASE_HEADERS,
         Authorization: `Bearer ${session.accessToken}`,
         'api-version': '2020-01-31',
       },
-      body: JSON.stringify(user),
+      body: JSON.stringify(group),
+    }).then(response => response.json())
```

Comment:
Does not return json - we should be checking on response.ok. This is all handled by the fetcher

Reply:
Fetcher will return truthily on success